origin.spec

```diff
@@ -299,7 +299,7 @@ for cmd in \
     openshift-router \
     kubectl
 do
-    ln -s oc %{buildroot}%{_bindir}/$cmd
+    ln -f -s oc %{buildroot}%{_bindir}/$cmd
```

Comment:
This is creating a link in %{_bindir} == `/usr/bin`

The e-mail complain I see is about this line failing:
`%{_datadir}/%{name}/linux/kubectl`
where %{_datadir} == `/usr/share`

What is creating those files in `%{buildroot}%{_datadir} ?

Reply:
Did you mean to include the right kubectl files back up about lines 278-280?

Reply:
i believe they are created by `make build` (line 227), you are right, I have to add them along with 'oc' in 278. Updated.